app/models/repo.js

```diff
@@ -148,6 +159,13 @@ const Repo = Model.extend({
       then(data => this._convertV3SettingsToV2(data['settings']));
   },
 
+  startMigration() {
+    const url = `/repo/${this.get('id')}/migrate`;
+    return this.get('api').post(url).then(() => {
```

Comment:
I can try removing the `.get`s, but we should probably aim for some consistency (at least on a file-by-file basis) to avoid confusion. You're right that we could use a task, I just didn't bother because we didn't need any derived state.